tools/pneumocat/pneumocat.xml

```diff
@@ -0,0 +1,80 @@
+<tool id="pneumocat" name="PneumoCaT" version="@VERSION@">
+    <description> Pneumococcal Capsular Typing of illumina reads</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+    
+    #import os
+
+    #if $input.type == "paired"
+        #set $for_input = "{}.fastq".format(os.path.splitext($input.forward.name)[0])
+        #set $rev_input = "{}.fastq".format(os.path.splitext($input.reverse.name)[0])
+        ln -s  "$input.forward" ./$for_input &&
+        ln -s  "$input.reverse" ./$rev_input &&
+
+    #else
+        #set $for_input = "{}.fastq".format(os.path.splitext($input.fastq_collection.forward.name)[0])
+        #set $rev_input = "{}.fastq".format(os.path.splitext($input.fastq_collection.reverse.name)[0])
+        ln -s "$input.fastq_collection.forward" $for_input &&
+        ln -s "$input.fastq_collection.reverse" $rev_input &&
+
+    #end if
+
+    PneumoCaT.py -1 "$for_input" -2 "$rev_input" -o outputs --threads "\${GALAXY_SLOTS:-1}" --cleanup
+
+    ]]></command>
+    <inputs>
+        <conditional name="input">
+            <param name="type" type="select" label="Sequence Data Type">
+                <option value="paired">Paired-end reads (FASTQ)</option>
+                <option value="paired_collection">Paired-end reads collection (FASTQ)</option>
+            </param>
+            <when value="paired">
+                <param name="forward"
+                    type="data" format="fastq,fastqsanger,fastq.gz,fastqsanger.gz"
+                    optional="false"
+                    label="Forward reads (FASTQ)"
+                    help="Must have ASCII encoded quality scores"
+                />
+                <param name="reverse"
+                    type="data" format="fastq,fastqsanger,fastq.gz,fastqsanger.gz"
+                    optional="false"
+                    label="Reverse reads (FASTQ)"
+                    help="File format must match the Forward FASTQ file"
+                />
+            </when>
+            <when value="paired_collection">
+                <param name="fastq_collection"
+                    type="data_collection" format="fastq,fastqsanger"
```

Comment:
why not gz here as well?

Reply:
Woops I missed that, I'll be sure to add it in and then add a test for that too potentially